pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -219,6 +219,9 @@ let
       # (we currently package 1.26 in Nixpkgs while Chromium bundles 1.21):
       # Source: https://bugs.chromium.org/p/angleproject/issues/detail?id=7582#c1
       ./patches/angle-wayland-include-protocol.patch
+      # Chromium reads initial_preferences from its own executable directory
+      # This patch modifies it to read /etc/chromium/initial_preferences
+      ./patches/chromium-initial-prefs.patch
```

Comment:
Due to the way how `electron` from source builds in nixpkgs depend on and reuse this `chromium` derivation, this addition to the `patches` list triggers not only a rebuild of `chromium` and `ungoogled-chromium`, but also to all `electron` builds and everything that depends on that.

This is fine and just something I wanted to note.